pkg/skaffold/deploy/helm_test.go

```diff
@@ -634,3 +565,83 @@ func makeRunContext(helmDeploy *latest.HelmDeploy, force bool) *runcontext.RunCo
 		},
 	}
 }
+
+func TestHelmGetManifestsFromReleases(t *testing.T) {
+	var tests = []struct {
+		description string
+		cmd         util.Command
+		runContext  *runcontext.RunContext
+		releases    []string
+		expected    kubectl.ManifestList
+	}{
+		{
+			description: "get manifests for multiple releases",
+			cmd:         &MockHelm{t: t},
+			runContext:  makeRunContext(testDeployWithMultipleReleases, false),
+			releases:    []string{"release-1", "release-2"},
+			expected: kubectl.ManifestList{
+				[]byte(serviceYaml),
+				[]byte(deploymentYaml),
+				[]byte(podYaml),
+			},
+		},
+		{
+			description: "get manifests for single release",
+			cmd:         &MockHelm{t: t},
+			runContext:  makeRunContext(testDeployConfig, false),
+			releases:    []string{"release-1"},
+			expected: kubectl.ManifestList{
+				[]byte(serviceYaml),
+				[]byte(deploymentYaml),
+			},
+		},
+		{
+			description: "get manifests for non existing release",
+			cmd:         &MockHelm{t: t},
+			runContext:  makeRunContext(testDeployWithMultipleReleases, false),
+			releases:    []string{"does-not-exist"},
+			expected:    kubectl.ManifestList{},
+		},
+		{
+			description: "get manifests for no releases",
+			cmd:         &MockHelm{t: t},
+			runContext:  makeRunContext(testDeployWithMultipleReleases, false),
+			releases:    []string{""},
+			expected:    kubectl.ManifestList{},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			event.InitializeState(tt.runContext)
+			defer func(c util.Command) { util.DefaultExecCommand = c }(util.DefaultExecCommand)
+			util.DefaultExecCommand = tt.cmd
+			tt.cmd.(*MockHelm).getResultBytes = mockReleseInfo
+			h := NewHelmDeployer(tt.runContext)
+			actual := h.getManifestsFromReleases(context.Background(), tt.releases)
+			testutil.CheckDeepEqual(t, tt.expected.String(), actual.String())
+		})
+	}
+
+}
+
+var (
+	releaseMap = map[string]string{
+		"release-1": fmt.Sprintf(`---
+# Source: skaffold-helm/templates/service.yaml
+%s---
+# Source: skaffold-helm/templates/deployment.yaml
+%s---`, serviceYaml, deploymentYaml),
+		"release-2": fmt.Sprintf(`---
+		# Source: skaffold-helm/templates/service.yaml
```

Comment:
This got me quite confused during review
```suggestion
		# Source: skaffold-helm/templates/pod.yaml
```

Reply:
will do

Reply:
done